easybuild/easyconfigs/g/glproto/glproto-1.4.17-intel-2016a.eb

```diff
@@ -12,6 +12,8 @@ toolchainopts = {'optarch': True}
 sources = [SOURCE_TAR_GZ]
 source_urls = [XORG_PROTO_SOURCE]
 
+builddependencies = [('xorg-macros', '1.19.0')]
```

Comment:
preferably, no. It's manually copying. My goal with this PR is to have a solid base to build Mesa 12 with 2016b for both intel and foss.